### PR TITLE
Add DB Connector release notes 1.3.1

### PR DESCRIPTION
**Pull request by estebanwasinger**

### CODE REVIEW

File: release-notes/v/latest/connector-db.adoc [hunk 1]
```diff
@@ -2,6 +2,35 @@
...
+
+* When doing a *Select* now the output metadata communicates correctly if the
+columns values can be have a null value or not.
+* In some scenarios when an error occurs the connection string with embedded
+credentials could be logged.
+* Now the Oracle Connection can be configured with a *Service Name*.
+
+=== Known Limitations and Workarounds
+
+None
```
Comment (marianogonzalez) [inline release-notes/v/latest/connector-db.adoc]: min mule version ?

Comment (estebanwasinger) [reply release-notes/v/latest/connector-db.adoc]: added

File: release-notes/v/latest/connector-db.adoc [hunk 2]
```diff
@@ -2,15 +2,50 @@
...
+
+=== New Features and Enhancements
+
+This release includes the following enhancements, changes, and fixes:
+
+* When doing a *Select* now the output metadata communicates correctly if the
+columns values can be have a null value or not.
+* In some scenarios when an error occurs the connection string with embedded
+credentials could be logged.
+* Now the Oracle Connection can be configured with a *Service Name*.
```
Comment (marianogonzalez) [inline release-notes/v/latest/connector-db.adoc]: this is the only enhancement. The other sare bug fixes

File: release-notes/v/latest/connector-db.adoc [hunk 3]
```diff
@@ -2,15 +2,50 @@
...
+None
+
+=== Fixed Issues
+
+[%header,cols="15a,85a"]
+|===
+|Issue |Description
+| https://www.mulesoft.org/jira/browse/MULE-14701[MULE-14701] | Connectors doesn't communicate correctly if their POJOs properties are nulleable or not
+| https://www.mulesoft.org/jira/browse/MULE-14740[MULE-14740] | Credentials leak in error messages
+| SE-1791 | Add the capability of configure the Service Name in a Oracle Connection
```
Comment (marianogonzalez) [inline release-notes/v/latest/connector-db.adoc]: this doesn't belong here, and RN cannot point to a private jira project

File: release-notes/v/latest/connector-db.adoc (hunk 2 above)
Comment (estebanwasinger) [reply release-notes/v/latest/connector-db.adoc]: All release notes have the structure: `This release includes the following enhancements, changes, and fixes`

**Pull request merged**